### DISCUSSION TITLE
Redpanda WASM magic ✨

### DISCUSSION THREAD

**Discussion by l0kr** [MaterializeInc/mz-hack-day-2022#8] [Show and tell]
My attempt to add WASM transformation to a project 👉 https://github.com/co0lster/mz-hack-day-2022/
Needs some docker work to get it running properly, but I will complete it in the future for sure.

Thank you very much for the event! I wanted to try these technologies for way too long. As a person that didn't know what is exactly dbt or how Materialize works, I am very thankful for the event and pretty surprised by how powerful these tools are. Can't way to hack some side projects with them!

**Comment by morsapaes** [comment]
Thanks again for joining us, @co0lster! It's great to know you managed to get into it not having that much experience with Materialize and dbt. Keep us posted as you ramp up!

[WASM transforms were also my bet, though I'll need a post-Hack Day Hack Day to work on it. 👯]
